Okta.Core.Tests/Clients/UsersClientTests.cs

```diff
@@ -509,13 +509,12 @@ public void SetCredentials()
                     Models.User updatedUser = usersClient.SetCredentials(strUserID, loginCreds);
 
                     Assert.IsNotNull(updatedUser, "The updated user is null so there likely was an error while updating his password.");
-
                 }
             }
             catch (OktaException e)
             {
                 string strEx = string.Format("Error Code: {0} - Summary: {1} - Message: {2}", e.ErrorCode, e.ErrorSummary, e.InnerException.InnerException.Message);
-                //Console.WriteLine(strEx);		
+                Console.WriteLine(strEx);		

```

Comment:
As in @marksztainbok-okta's comment, we shouldn't be writing it, we should be throwing it if we get into a bad state.